### ISSUE TITLE
Improve test vector ergonomics

### ISSUE THREAD

**Issue by iand** [filecoin-project/lily#651]
### Description

Creating test vectors is a time consuming process even with the tooling we have created and needs to be improved so we can use and create vectors more often:

 - To recreate an existing vector after a bug fix requires finding a lotus repository containing the state for the heights in the vector or running the vector command against a running full archival node. 
 - The tooling doesn't work with a lily node.
 - The vector files are huge and difficult to work with
 - Failures in CI are obscure and hard to parse because of the size of the files

Suggestions:

 - use calibration net for more manageable data sizes
 - allow the base state to be specified with the cid of a full state car file
 - change vector command to spin up a temporary lily node and import the above car file so tests can be run against it or new vectors created easily

**Comment by iand** [comment]
See also https://github.com/filecoin-project/sentinel-visor/issues/603

**Comment by frrist** [comment]
Addressed via: https://github.com/filecoin-project/lily/pull/811
- uses calibration net, but supports any network there is a car file and genesis for.
- spins up lily node in memory and performs walk.
